src/app/shared/components/navigation/navigation.component.scss

```diff
@@ -1,10 +1,11 @@
-:host {
+@import "src/styles/colors";
 
+:host {
   .navbar {
-    background-color: #99c435;
+    background-color: $yellow-green-green;
     height: 120px;
-    box-shadow: 2px 2px 10px #397207;
-    text-shadow: 2px 2px 3px #307207;
+    box-shadow: 2px 2px 10px green;
+    text-shadow: 2px 2px 3px green;
```

Comment:
All colors of the Application should be declared in the single place ("src/styles/colors"). 
So don't use native "green" color here, but declare corresponding to native "green" #hash color in "src/styles/colors" and reference it here.

Reply:
Fixed 'green' and other colors declarations and references. .